pybsn/test/test_switch.py

```diff
@@ -0,0 +1,56 @@
+import os
+from betamax import Betamax
+from requests import Session
+from nose import with_setup
+from ..api import Api
+
+with Betamax.configure() as config:
+    config.cassette_library_dir = 'fixtures/cassettes/switch'
+
+
+def test_get_switches():
+	api = Api(os.environ['PYBSN_HOST'], os.environ['PYBSN_USER'], os.environ['PYBSN_PASS'])
```

Comment:
So we need these values from the environment in case we're doing a VCR recording? Can we default them to 127.0.0.1/etc?